src/editortoolkit_cmn.cpp

```diff
@@ -213,7 +213,6 @@ bool EditorToolkitCMN::Delete(std::string &elementId)
 {
     Object *element = this->GetElement(elementId);
     if (!element) return false;
-
```

Comment:
It's generally better if you don't change the existing formatting unless there's a reason. Removing the blank line here is innocent enough, but it adds additional unnecssary changes to the diff.

Reply:
Personally I liked that space there.  I just received guff about having the gap there

Reply:
"guff" from  me? 

I was commenting on the additional spaces (trailing / hanging spaces) that had been added to the line, not to the blank line itself. Sorry if I wasn't clear about that.